tests/test_dle.py

```diff
@@ -81,46 +81,23 @@ def test_composite_transformation(shape):
     assert np.equal(wo_blocking.data, w_blocking.data).all()
 
 
-@pytest.mark.parametrize("blockinner,exp_calls,exp_iters", [
-    (False, 4, 5),
-    (True, 8, 6)
+@pytest.mark.parametrize("blockinner, openmp", [
+    (False, True), (False, True)
 ])
-def test_cache_blocking_structure(blockinner, exp_calls, exp_iters):
+def test_cache_blocking_structure(blockinner, openmp):
     # Check code structure
     _, op = _new_operator2((10, 31, 45), time_order=2,
-                           opt=('blocking', {'blockinner': blockinner,
-                                             'par-collapse-ncores': 1}))
-    calls = FindNodes(Call).visit(op)
-    assert len(calls) == exp_calls
-    trees = retrieve_iteration_tree(op._func_table['bf0'].root)
-    assert len(trees) == 1
-    tree = trees[0]
-    assert len(tree) == exp_iters
+                           opt=('blocking', {'openmp': True, 'blockinner': blockinner,
```

Comment:
`openmp`: openmp?

Reply:
nitpicking: blockinner could probably be moved to `parametrize` as well

Reply:
WDYM, it is alreday there?